src/pal/prebuilt/inc/xclrdata.h

```diff
@@ -3333,7 +3333,7 @@ enum __MIDL___MIDL_itf_xclrdata_0000_0008_0005
     {
         CLRDATA_METHNOTIFY_NONE	= 0,
         CLRDATA_METHNOTIFY_GENERATED	= 0x1,
-        CLRDATA_METHNOTIFY_DISCARDED	= 0x2
+        CLRDATA_METHNOTIFY_PITCHED	= 0x2
```

Comment:
This file might be regenerated in the future from an idl we have on desktop. Its probably better to treat this as a public contract we don't change.